pages/map.js

```diff
@@ -177,78 +198,117 @@ export default function Map() {
-        </div>
+        </Box>
 
-        <div className={styles.floatingContainer}>
-          <button className={styles.locationButton} onClick={handleSearchGeo}>
+        <Flex
+          position="absolute"
+          bottom="1rem"
+          left="1rem"
+          right="1rem"
+          flexDir="column"
+        >
+          <Button
+            alignSelf="flex-end"
+            justifySelf="flex-end"
+            background="white"
+            padding="1rem"
+            boxShadow="0 2px 8px 0 rgb(48 49 53 / 16%)"
+            borderRadius="8px"
+            marginBottom="1rem"
+            onClick={handleSearchGeo}
+          >
             📍
-          </button>
-          <div
-            className={styles.hospitalInfoWrapper}
+          </Button>
+          <Box
+            borderRadius="8px"
+            padding="1rem"
+            boxShadow="0 2px 8px 0 rgb(48 49 53 / 16%)"
+            background="white"
             onClick={(e) => {
               e.preventDefault()
               setPopupHospitalVisibility(true)
             }}
           >
-            <p onClick={() => setPopupHospitalVisibility(true)}>
+            <Text onClick={() => setPopupHospitalVisibility(true)}>
               Jumlah Rumah Sakit:{' '}
               {isLoading ? <Spinner /> : hospitalList?.length}{' '}
               <span style={{ color: '#F87A26', cursor: 'pointer' }}>
                 (Daftar Rumah Sakit)
               </span>
-            </p>
-          </div>
-        </div>
-      </div>
+            </Text>
+          </Box>
+        </Flex>
+      </Box>
 
       <BottomSheet
         open={popupHospital}
         onDismiss={() => setPopupHospitalVisibility(false)}
       >
-        <div style={{ padding: '1rem', color: 'black' }}>
-          {hospitalList?.map((hospital) => (
-            <div
-              style={{
-                padding: '.5rem 0',
-                borderBottom: '1px solid #EAEAEA',
-              }}
-              key={hospital.name}
-              onClick={() => {
-                setPopupHospitalVisibility(false)
-                map.current.flyTo({
-                  center: [hospital.lon, hospital.lat],
-                  zoom: 12,
-                })
-              }}
-            >
-              <p style={{ fontWeight: 'bold', marginBottom: '.5rem' }}>
-                {hospital.name}
-              </p>
-              <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-                <p>Tempat tidur tersedia: {hospital.available_bed}</p>
-                <p>
-                  {hospital.bed_queue
-                    ? `${hospital.bed_queue} antrian`
-                    : 'Tanpa antrian'}
-                </p>
-              </div>
-              <p>{hospital.address}</p>
-            </div>
-          ))}
-        </div>
+        <Box padding="1rem" color="black">
+          <VStack align="start" spacing="4">
+            {!isLoading ? (
+              hospitalList.map((hospital) => (
```

Comment:
Please filter this too, so the data is more aligned with the map data. 🙇🏻

Reply:
hoo, okay 😅

Reply:
Or make some text info like, "`x` rumah sakit tersedia dari `x`", that would be great too! 🙇🏻

Reply:
@agallio  So, we'll still show thee not available hospital in the hospital list, right?

Reply:
Yep, I think so. So I guess just modify the bottom card wording ya

Reply:
Map only render available hospital, bottom sheet render all hospital, floating card showing the number available hospital & all hospital ✅

Reply:
Very niceeeeeee! 🙇🏻